src/transform/adm0.py

```diff
@@ -49,6 +54,27 @@ def transform(source: str, input_filename: str, schema_filename: str, output_fil
         schema_mapping = {
             'NAME_0': 'name_en'
         }
+    elif source == "geoboundaries":
+        rawdir = config['dirs']['raw_data']
+        source_geob = os.path.join(rawdir, config['geoboundaries']['adm0']['raw'])
```

Comment:
you don't need to use the config file here, this filename should have been passed in as `input_filename`